Tasks/DotNetCoreInstallerV0/task.json

```diff
@@ -32,14 +40,32 @@
                 "runtime": "Runtime",
                 "sdk": "SDK (contains runtime)"
             }
+        },        
+        {
+            "name": "useGlobalJson",
+            "type": "boolean",
+            "label": "use global.json",
+            "defaultValue": false,
+            "required": false,
+            "helpMarkDown": "If you set this box, the task will search for all global.json from your working directory and install the matching SDK.",
+            "visibleRule": "packageType = sdk"
         },
         {
             "name": "version",
             "type": "string",
             "label": "Version",
             "defaultValue": "1.0.4",
-            "required": true,
-            "helpMarkDown": "Specify exact version of .NET Core SDK or runtime to install.<br/>Find the value of `version-sdk` for installing SDK, or `version-runtime` for installing Runtime from any releases [here](https://github.com/dotnet/core/blob/master/release-notes/releases.json)"
+            "required": false,
```

Comment:
It should be required when visible.

Reply:
How can we get it only be required when visible?

Reply:
You can keep `"required" : true`, and keep the visible rule in place. When the input is visible, the UI will make sure that `version` input is required.

Reply:
I think we have resolve this. If not reopen this conversion.